Src/FluentAssertions/Primitives/StringAssertions.cs

```diff
@@ -660,7 +693,40 @@ public AndConstraint<StringAssertions> ContainEquivalentOf(string expected, stri
             Execute.Assertion
                 .ForCondition(Contains(Subject, expected, StringComparison.CurrentCultureIgnoreCase))
                 .BecauseOf(because, becauseArgs)
-                .FailWith("Expected {context:string} to contain equivalent of {0}{reason} but found {1}.", expected, Subject);
+                .FailWith("Expected {context:string} {0} to contain equivalent of {1}{reason}, but not found.", Subject, expected);
```

Comment:
That's a weird failure message. Sloppy paste?

Reply:
This is something left over from the original PR. Looking at it, I don't see why the message was changed at all in the first place.

Reply:
Ok, so the messages for `Contain` and `ContainEquivalentOf` are not consistent, so the original author probably decided to make them the same, but ended up with a weird mixture of both.

The choice here is between the following 2 options
1. "Expected string to contain "XYZ" because that is required but found "ABCDEF"
2. "Expected string "ABCDEF" to contain "XYZ" because that is required."

In my opinion the second option sounds more clear and natural, so I'd go with that personally.

Reply:
I agree